components/src/dynamo/profiler/utils/model_info.py

```
@@ -112,6 +116,48 @@ def get_model_weight_size(
+                return False
+        else:
+            config_path = Path(
+                hf_hub_download(
+                    repo_id=str(model_name_or_path),
+                    filename="config.json",
+                    token=token,
+                )
+            )
+        with open(config_path) as f:
+            cfg = json.load(f)
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)